src/App.js

```diff
@@ -1,29 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/button-has-type */
```

Comment:
- [ ] I suggest you add a type attribute to the button element instead of disabling this rule. By specifying either `button`, `submit` or `reset` (in this case button), the code's purpose is clear and is easier to maintain